net-irc/atheme-services/atheme-services-7.2.9.ebuild

```diff
@@ -0,0 +1,116 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils flag-o-matic perl-module user
+
+DESCRIPTION="A portable and secure set of open-source and modular IRC services"
+HOMEPAGE="https://github.com/atheme/atheme"
+SRC_URI="https://github.com/atheme/atheme/releases/download/v${PV}/atheme-${PV}.tar.bz2 -> ${P}.tar.bz2"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~x86-fbsd ~amd64-linux"
+IUSE="cracklib largenet ldap nls +pcre perl profile ssl"
+
+RDEPEND=">=dev-libs/libmowgli-2.1.0:2
+	cracklib? ( sys-libs/cracklib )
+	ldap? ( net-nds/openldap )
+	nls? ( sys-devel/gettext )
+	perl? ( dev-lang/perl )
+	pcre? ( dev-libs/libpcre )
+	ssl? ( dev-libs/openssl:* )"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+pkg_setup() {
+	# the dependency calculation puts all of the .c files together and
+	# overwhelms cc1 with this flag :-(
+	filter-flags -combine
```

Comment:
No idea.  I tried commenting out the 'filter-flags -combine' option and it compiled fine for me.